proxy/http/Http1ClientSession.cc

```diff
@@ -261,8 +261,8 @@ Http1ClientSession::do_io_close(int alerrno)
   } else {
     HttpSsnDebug("[%" PRId64 "] session closed", con_id);
     HTTP_SUM_DYN_STAT(http_transactions_per_client_con, transact_count);
-    if (read_state != HCS_ACTIVE_READER) {
-      // donot double decrement
+    if (conn_decrease) {
```

Comment:
We are increasing the stat in `new_connection()`, so I think it should be sufficient that we decrease the stat in `free()`.
Could you help me understand what is the reason that we have it in `do_io_close()`?

Reply:
That is true. Should be sufficient to just leave it in the free.  Let me try that out.